src/sources/polymarket.py

```
@@ -228,40 +228,49 @@ def update(
             "ascending": False,
+            "liquidity_num_min": _MIN_MARKET_LIQUIDITY,
         }
 
         while True:
-            params["offset"] = offset
+            if after_cursor:
+                params["after_cursor"] = after_cursor
             try:
-                logger.info(f"Fetching markets with offset {offset}.")
+                logger.info(f"Fetching markets (cursor={after_cursor}).")
                 response = requests.get(_GAMMA_API_URL, params=params)
```

Ruff (0.15.20)

[error] Probable use of `requests` call without timeout

(S113)